frontend/src/guidance/AdditionalFindings.js

```diff
@@ -85,6 +87,67 @@ export function AdditionalFindings({ domain }) {
           <Trans>What are these additional findings?</Trans>
         </Button>
         <Accordion allowMultiple defaultIndex={[0, 1, 2, 3, 4, 5]} w="100%">
+          <AccordionItem id="vulnerabilities">
```

Comment:
Looks like chakra is stripping the id here. Can wrap this section with a `<Box id="vulnerabilities">` to get the scrolling/outline to work though.